app/components/primer/button_component.rb

```diff
@@ -78,8 +78,8 @@ class ButtonComponent < Primer::Component
     #
     # @param scheme [Symbol] <%= one_of(Primer::ButtonComponent::SCHEME_OPTIONS) %>
     # @param variant [Symbol] <%= one_of(Primer::ButtonComponent::VARIANT_OPTIONS) %>
-    # @param tag [Symbol] <%= one_of(Primer::BaseButton::TAG_OPTIONS) %>
-    # @param type [Symbol] <%= one_of(Primer::BaseButton::TYPE_OPTIONS) %>
+    # @param tag [Symbol] (Primer::BaseButton::DEFAULT_TAG) <%= one_of(Primer::BaseButton::TAG_OPTIONS) %>
```

Comment:
What do you think about evaluating these constants to get their value?

Reply:
the `docs.rake` takes care of that 😉 
![image](https://user-images.githubusercontent.com/11280312/121416240-316f7a00-c92e-11eb-8169-4143a159dc36.png)